src/Catty/Defines/LanguageTranslationDefinesSwift.swift

```diff
@@ -34,6 +34,8 @@
                    |___/             |___/
  */
 
+/* { PLACEHOLDER HEADER, DO NOT DELETE THIS LINE (used in synchronize_localization.py) } */
```

Comment:
Everything works fine. One suggestion would be to remove this placeholder header and always insert the Swift code at the end of this file. Could you please check if that works?

Reply:
Great suggestion, this way it is cleaner. I've just pushed the changes.